Library/Homebrew/dev-cmd/pr-upload.rb

```diff
@@ -90,7 +90,7 @@ def bottles_hash_from_json_files(json_files, args)
   def pr_upload
     args = pr_upload_args.parse
 
-    # odeprecated "`brew pr-upload --github-org`", "`brew pr-upload` without `--github-org`" if args.github_org
+    odisabled "`brew pr-upload --github-org`", "`brew pr-upload` without `--github-org`" if args.github_org
```

Comment:
Use the `replacement` feature of the CLI parser instead so it gets hidden from documentation.

(I'm assuming it's intention to go directly to `odisabled` here.)

Reply:
@Bo98 Already hidden but not intended to go straight to `odisabled`, fixed, thanks.

Reply:
You also can go and remove all the existing args that have `replacement:`.

Reply:
@Bo98 Good shout, done and added to docs too.